### PR TITLE
More proc additions

### PR DESCRIPTION
**Pull request by Dennisbonke**
This time I decided to tackle some low hanging fruits in `/proc/sys/kernel`, so here are `ostype` (Managarm), `osrelease` (0.0.1) and `arch` (set to either `x86_64` or `AArch64` depending on architecture).

### CODE REVIEW

File: posix/subsystem/src/procfs.cpp [hunk 1]
```diff
@@ -358,6 +367,54 @@ async::result<void> UptimeNode::store(std::string) {
...
+async::result<std::string> ArchNode::show() {
+	// See man 5 proc for more details.
+	// Based on the man page from Linux man-pages 6.01, updated on 2022-10-09.
+	std::stringstream stream;
+#if defined(__x86_64__)
+	stream << "x86_64\n";
+#elif defined(__aarch64__)
+	stream << "AArch64\n";
+#else
+	#error "Unknown architecture"
```
Comment (ElectrodeYT) [inline posix/subsystem/src/procfs.cpp:407]: Maybe we could add a riscv entry here? We dont support it yet, but there is no harm in adding it now

`#if defined(__riscv) && __riscv_xlen == 64`

File: posix/subsystem/src/procfs.cpp [hunk 2]
```diff
@@ -358,6 +367,54 @@ async::result<void> UptimeNode::store(std::string) {
...
+	// TODO: proper error reporting.
+	std::cout << "posix: Can't store to a /proc/sys/kernel/ostype file" << std::endl;
+	co_return;
+}
+
+async::result<std::string> OsreleaseNode::show() {
+	// See man 5 proc for more details.
+	// Based on the man page from Linux man-pages 6.01, updated on 2022-10-09.
+	std::stringstream stream;
+	stream << "0.0.1\n";
```
Comment (ElectrodeYT) [inline posix/subsystem/src/procfs.cpp:388]: I'm not well versed in this, is there a standard response for a git or rolling version, or do we have to report a "real" version here?

**Review by ElectrodeYT — APPROVED** [review]
LGTM, but a couple questions

File: posix/subsystem/src/procfs.cpp (hunk 2 above)
Comment (Dennisbonke) [reply posix/subsystem/src/procfs.cpp:388]: We don't have a version really, so placeholder I guess

File: posix/subsystem/src/procfs.cpp (hunk 1 above)
Comment (Dennisbonke) [reply posix/subsystem/src/procfs.cpp:407]: Noted, will add

Comment (Dennisbonke) [reply posix/subsystem/src/procfs.cpp:410]: Fixed

File: posix/subsystem/src/procfs.cpp (hunk 2 above)
Comment (Dennisbonke) [reply posix/subsystem/src/procfs.cpp:389]: Fixed

**Review by no92 — APPROVED** [review]
LGTM

**Pull request merged**